test/functional/sandbox/js/helper.js

```diff
@@ -120,6 +120,19 @@
     }
   });
 
+  router.register("/C2592", "event-command", {
```

Comment:
I might be reading this wrong, but it seems like the other routes in this file are set up to be potentially useable by more than a single test. i think it would be useful to do the same here. In other words, instead of naming this route `/C2592`, maybe name it `/generic-event-with-xdm` or something like that.

Actually, scratch that. There's already an `/event` route--the only difference is that it sends generic data (using the `data` option) rather than xdm (using the `xdm` option), but none of the tests that are using the `event` route are testing anything specific to generic data yet. I would recommend just changing `data` to `xdm` in the `/event` route and using that route in your test. If we ever need to test sending generic data (using the `data` option), we an add that later.